implementations/rust/ockam/ockam_command/src/vault/create.rs

```diff
@@ -27,12 +31,22 @@ async fn run_impl(
     ctx: Context,
     (options, cmd): (CommandGlobalOpts, CreateCommand),
 ) -> crate::Result<()> {
-    let mut rpc = Rpc::background(&ctx, &options, &cmd.node_opts.api_node)?;
-    let request = Request::post("/node/vault").body(CreateVaultRequest::new(cmd.path));
+    if cmd.node_opts.is_some() {
+        let node_name = &cmd.node_opts.unwrap().api_node.clone();
```

Comment:
You can write this as follows to avoid the `unwrap`:
if let Some(node_opts) = cmd.node_opts {
    ....
}